config/crd/apiextensions.k8s.io/v1/backup.appuio.ch_prebackuppods.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
   creationTimestamp: null
   name: prebackuppods.backup.appuio.ch
 spec:
+  preserveUnknownFields: false
```

Comment:
Just to double check: You've meant to set those to `false`, right?

Reply:
Yes. If it's set to true, then the cluster wouldn't accept the CRD when applying with kubectl, as described in the linked issue.